README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the Ani's portfolio and SPS projects.
```

Comment:
Remove the "the" in the sentence and it's good!

"This repo contains Ani's portfolio and SPS projects."

Reply:
Good Catch, thank you!